.github/workflows/build.yml

```diff
@@ -115,6 +115,14 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+      - name: Docker test
```

Comment:
Just a thought: we might add another dimension to a matrix build, so that the alpine and "default" variants wouldn't need to be copied (and maintained) in seperate build steps. Yet, we might first add the Windows variant and see how we could merge the workflows in a more consistent manner.